tests/rules/test_category.py

```diff
@@ -132,3 +132,29 @@ def test_get_categories(data, max_uniques, expected_stats, expected_message):
     assert result == create_result(
         "Categories", {Level.INFO: [(expected_message,)]}, stats=expected_stats
     )
+
+
+@pytest.mark.parametrize(
```

Comment:
Probably, it would be nice to add at least one more test case where it does not find any categorical column.
Also, not sure if it would be helpful to make some kind of check in this method.
Say that I allow `max_uniques=100` and I set `sample=100` then `range(100)` is valid, although it might not be.
So, my thought here is that the nearer `max_uniques / sample` is to zero, the better.

Reply:
>Say that I allow max_uniques=100 and I set sample=100 then range(100) is valid, although it might not be.

I think if someone passes those arguments, he knows what he is doing. From the point of this function you are right, but right now it server the only purpose - get the list of categorical columns very fast.

Reply:
Agreed @manycoding .
My idea was to log something. But we can think about it later (if this kind of scenario appears).